types/node/v12/http.d.ts

```diff
@@ -116,6 +116,96 @@ declare module "http" {
          */
         headersTimeout: number;
         keepAliveTimeout: number;
+
+        // From http.server
+        addListener(event: 'checkContinue', listener: RequestListener): this;
```

Comment:
Please add these to the latest node version as well, not just v12.

Reply:
(The signature of v13 seems not correct, missing headersTimeout, keepAlive etc...)
I added the events: 0966b3b